app/main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        self.frontend_skills = ["JavaScript", "HTML", "CSS"]
+        super().__init__(name)
+        for skill in self.frontend_skills:
+            self.learn_skill(skill)
```

Comment:
Use the extend() method or += operator to add multiple values to a single list. Read checklist

Reply:
fix in all places